Makefile

```diff
@@ -61,7 +61,6 @@ requirements: requirements/requirements.txt
 .PHONY: test
 test:
 	pytest -n auto -v --cov=mitiq --cov-report=term --cov-report=xml --ignore=mitiq/interface/mitiq_pyquil
-
```

Comment:
Let's leave the spacing between the makefile commands. It makes the file a touch easier to read.

Reply:
Thanks for catching this! I shouldn't have made the change.